src/config/external-applications.md

```diff
@@ -82,3 +82,61 @@ example, to build the control package, you must install the `openblas-devel`,
 
 To use MATLAB's help browser, live scripts, add-on installer, and simulink,
 install the `libselinux` package.
+
+## Steam
+
+### Installing Steam through Flatpak
+
+Steam can be most easily be installed via [Flatpak](##flatpak). If you are using
+a different drive to store you game library run the following commmand, where
+`<storage>` is the mount point of your drive:
+
+`flatpak override com.valvesoftware.Steam --filesystem=<storage>`
+
+If sound is not working when Steam is installed via Flatpak, creating and
+running the following script to start Steam may fix the issue:
+
+```
+#!/bin/sh
+pulseaudio --start flatpak run com.valvesoftware.Steam
+```
+
+### Installing Steam through XBPS
+
+To install Steam you first need to [enable the "nonfree"
+repository](../xbps/repositories/index.md###nonfree). Then, install the `steam`
+package; this should be enough for i686 systems. On x86_64 systems, additional
+32-bit libraries must be installed. These libraries are listed upon installing
+the `steam` package, and can be viewed in [Steam's install
+message](https://github.com/void-linux/void-packages/blob/master/srcpkgs/steam/INSTALL.msg).
+You will need to [enable](../config/services/index.md####enabling-services) the
```

Comment:
Kind of unrelated, but I think we could add a tiny item to the XBPS part about seeing install msgs (`xbps-query -p install-msg <package>`), and this would link there instead of to the git repo.